### DISCUSSION TITLE
Column-level edit permissions

### DISCUSSION THREAD

**Discussion by noamross** [nocodb/nocodb#670] [Features]
Related to #640, I was wondering if column-level permissions are on your roadmap.  One application we use this in with AirTable is to have a "validated" checkoff/boolean column that only certain users can edit, which lets us to QA/QC

**Comment by o1lab** [comment]
Which is the application that you use with Airtable ? 

Column level edit are there in nocodb but currently are tucked away ( in API ACL - you can click on column name (header) and you should be able to set additional rules - but they won't work yet ) . To make them work with nested tables - we've to rethink a bit.
